lib/modules/userTagger.js

```diff
@@ -59,17 +53,25 @@ module.options = {
 		description: 'By default, store a link to the comments when tagging a user in a link post. Otherwise, the link (that the post refers to) will be used.',
 		advanced: true,
 	},
+	colorUser: {
```

Comment:
Or add a separate option called "trackUserWeight" and make all three of these depend on it. And then close my issue about. It.